### PR TITLE
update version in package.json and cargo.toml to 0.20.1

### PR DESCRIPTION
**Pull request by rllola**
Update the version info of the package to `0.20.1` next tag.

### CODE REVIEW

**Comment by jleni** [comment]
why do we need this?
GitHub actions theoretically will use the release version and override these values just before publishing

**Comment by rllola** [comment]
>  GitHub actions theoretically will use the release version and override these values just before publishing

I didn't realize that github action was adding it automatically at release. It does feel weird because the current version values can be misleading. Should I just put `0.0.0` as a placeholder ?

**Comment by jleni** [comment]
> > GitHub actions theoretically will use the release version and override these values just before publishing
> 
> I didn't realize that github action was adding it automatically at release. It does feel weird because the current version values can be misleading. Should I just put `0.0.0` as a placeholder ?

yes, I think it makes more sense to have 0.0.0 there

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "filecoin-signing-tools",
   "private": true,
   "description": "Filecoin signing tools",
-  "version": "0.1.0",
+  "version": "0.20.1",
```
Comment (jleni) [inline package.json:5]:
```suggestion
  "version": "0.0.0",
```

File: signer-npm/package-signer-wasm-browser.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "Zondax <info@zondax.ch>"
   ],
   "description": "",
-  "version": "0.1.0",
+  "version": "0.20.1",
```
Comment (jleni) [inline signer-npm/package-signer-wasm-browser.json:7]:
```suggestion
  "version": "0.0.0",
```

File: signer-npm/package-signer.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "cryptography"
   ],
   "description": "Filecoin Signing Tools",
-  "version": "0.2.0",
+  "version": "0.20.1",
```
Comment (jleni) [inline signer-npm/package-signer.json:12]:
```suggestion
  "version": "0.0.0",
```

**Review by jleni — CHANGES_REQUESTED** [review]

**Pull request merged**